db/db_utils.py

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
could you do some clean up of the file @MeenuV? (imports are one line, alphabetize) 
separate first and third party libs
MagpieData does not need a row count attribute since it's always 1 or 0, None on data is fine
MagpieDbError should be put into exceptions.py
MagpieData should be in data.py (decouple db logic)

I think we should name the 'metadata' attribute in our table 'json_metadata' instead